src/content/guides/requesting-pre-auth.mdx

```diff
@@ -19,65 +19,44 @@ Pre Auth payments are not supported in all cases.
 
 Pre Auth payments go through an orthogonal payment flow compared to Centrapay’s standard payment flow.
 
+> The example below shows the integrator polling for the Payment Request, this is not needed for eCommerce flows as the integrator will be informed when the authorization has completed.
+
 > See also: [Requesting Payment](/guides/requesting-payment).
 
 ```mermaid
 sequenceDiagram
 	autonumber
 
 	participant Patron
-	participant POS
+	participant Integrator
 	participant Centrapay
 
-	note over Patron, POS: Connect with Patron
-	POS->>Centrapay: Create Authorization
+	note over Patron, Integrator: Connect with Patron
+	Integrator->>Centrapay: Create Authorization
 
-	note over POS: Poll for Payment Confirmation
+	note over Integrator: Poll for Payment Confirmation
 
-	note over POS: ✅ Display Successful Authorization
+	note over Integrator: ✅ Display Successful Authorization
 
-	note over POS, Centrapay: ⏱️ Some time later..
+	note over Integrator, Centrapay: ⏱️ Some time later..
 
 	loop 0..Many (Up to Authorized Amount)
-		POS->>Centrapay: Make Confirmations Against Authorized Funds
-		note over POS: ✅ Display Successful Confirmation
+		Integrator->>Centrapay: Make Confirmations Against Authorized Funds
 	end
 
-	POS->>Centrapay: Release Remaining Authorized Funds
-	note over POS: ✅ Display Successful Release
+	Integrator->>Centrapay: Release Remaining Authorized Funds
 ```
 
-1. The POS places a hold on funds by creating an Authorization.
-2. The POS draws down on authorized funds by making Confirmations against the authorization when the purchase is ready to be fulfilled.
-3. The POS releases any remaining funds that have not been confirmed back to the Patron.
+1. The Integrator places a hold on funds by creating an Authorization.
+2. The Integrator draws down on authorized funds by making Confirmations against the authorization when the purchase is ready to be fulfilled.
+3. The Integrator releases any remaining funds that have not been confirmed back to the Patron.
 
 ### Authorize
 
 An authorization is created when the [Payment Request is created](/api/payment-requests#create-a-payment-request) with the `preAuth` flag while [Requesting Payment](/guides/requesting-payment).
 
 Once the authorization is successful, the Payment Request `preAuthStatus` is set to `authorized`.
 
-```mermaid
```

Comment:
These diagrams were just the above diagram sectioned out, I feel they are unnecessary and can add some confusion since they skip steps in the process.